client/src/components/BubblePage.js

```diff
@@ -1,17 +1,29 @@
 import React, { useState, useEffect } from "react";
-import axios from "axios";
+
+import axiosWithAuth from '../utils/axiosWithAuth'
+
+
 
 import Bubbles from "./Bubbles";
 import ColorList from "./ColorList";
 
+
 const BubblePage = () => {
   const [colorList, setColorList] = useState([]);
   // fetch your colors data from the server when the component mounts
   // set that data to the colorList state property
-
+  const token= localStorage.getItem(token)
+useEffect(()=>{
+ 
+  axiosWithAuth()
+  .get('/api/colors', token)
+    .then(res => {
```

Comment:
Our `axiosWithAuth()` function takes care of the token and local storage for us, so we don't need the token here. This code could just look like:
```
useEffect(() => {

    axiosWithAuth()
        .get('/api/colors')
        .then(res => more code here . . .
```